.coderabbit.yaml

```
@@ -0,0 +1,21 @@
+    sequence_diagrams: true
+    auto_review:
+        enabled: true
+        ignore_title_keywords:
+            - "WIP"
+            - "DO NOT MERGE"
+        base_branches:
+            - "master"
+            - "development"
+            - "release/*"
+chat:
+    auto_reply: true
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)